js/bittrex.js

```diff
@@ -505,8 +505,18 @@ module.exports = class bittrex extends Exchange {
         if (typeof side === 'undefined')
             side = this.safeString (order, 'Type');
         let isBuyOrder = (side === 'LIMIT_BUY') || (side === 'BUY');
-        side = isBuyOrder ? 'buy' : 'sell';
-        let status = 'open';
+        let isSellOrder = (side === 'LIMIT_SELL') || (side === 'SELL');
+        if (isBuyOrder) {
+            side = 'buy';
+        }
+        if (isSellOrder) {
+            side = 'sell';
+        }
+        // We parse different fields in a very specific order.
+        // Order might well be closed and then canceled.
```

Comment:
Closed then canceled? o.O

Reply:
Not bittrex specifically (I have just verified) but we have seen that in other exchanges. 

Let's say you are canceling a closed order. The exchange can return either a fail (order is already closed) or success. After all if you like to cancel a closed order, why not. The only thing that changes in order is the status from 'closed' to 'cancelled' and nothing else. So now you have an order filled to 100% with `status == canceled`. As weird as it is it's still something that is logically possible. 

Another possible case: a race condition in the exchange code. The last chunk of order is filled just at the moment the order has been canceled. There you go: canceled an 100% filled at the same time. 

(And nope I don't remember why we have put this comment in this commit specifically :-(